Fixes wrong image bug on update feed

kumarpalsinh25 opened the pull request.

gnunicorn commented:
@kumarpalsinh25 I suppose there isn't anything we can do about the background flashing when swiping left and right?

kumarpalsinh25 commented:
> @kumarpalsinh25 I suppose there isn't anything we can do about the background flashing when swiping left and right?

I have figure out the flashing issue and fixed it.

Commit:
https://github.com/acterglobal/a3/pull/1868/commits/107794ab6c4f69525ab1038c75326bc0e227fba3

Reference Video:

https://github.com/acterglobal/a3/assets/51526480/782d12e8-bea7-4eab-8832-bcb79f3cf095

gnunicorn commented:
> I have figure out the flashing issue and fixed it.

But looking at your video the background doesn't properly update the color anymore. Check the video from before with the one you posted now: when sliding left and right the slide background stays the same though before there were different backgrounds per slide.

kumarpalsinh25 commented:
> @kumarpalsinh25
> 
> > I have figure out the flashing issue and fixed it.
> 
> But looking at your video the background doesn't update the color anymore, that's why there is no falshing. Check the video from before with the one you posted now: when sliding left and right the slide background stays the same though before there were different backgrounds per slide.

Flashing was not related to background color. I have fixed the background issue on below commit.
https://github.com/acterglobal/a3/pull/1868/commits/ffe059c6d0af191b8b99075289bb8a4aa2d7ef0e

Video Reference:

https://github.com/acterglobal/a3/assets/51526480/9c5baad7-11cf-4ae9-8d83-d312f9e999b1

gnunicorn commented on 1868-update-slide-wrong-image.md:
```diff
@@ -0,0 +1 @@
+- [Fixes] : Wrong image/video in update slide issue is resolved
```
this needs to go into `.changes/`

gnunicorn commented:
@kumarpalsinh25 oh, so nice! now only move the `.changes/` file and we are all good :D
